check_command.go

```diff
@@ -48,33 +75,42 @@ func (c *CheckCommand) Run(request CheckRequest) ([]Version, error) {
 			continue
 		}
 
-		if release.TagName == nil {
-			continue
-		}
-		if _, err := version.NewVersionFromString(versionParser.parse(*release.TagName)); err != nil {
-			continue
+		if orderByTime {
+			// We won't do anything with the tags, so just make sure the filter matches the tag.
+			var tag string
+			if release.TagName != nil { tag = *release.TagName }
```

Comment:
Minor nit: this is unconventional Go formatting; I would recommend running `go fmt` before committing (there's also usually text editor plugins to do this automatically on save).

Reply:
OK. Running go fmt also modifies a bunch of lines and files unrelated to this PR. Would you like me to commit all those changes?

Reply:
(Sorry for the delay, I was on vacation!)

Yeah, committing those is fine. Feel free to only commit the files you changed, but either way we want to converge on `go fmt`ed code over time.

Reply:
Done.